src/views/new-evaluation/newEvaluation.js

```diff
@@ -0,0 +1,302 @@
+    ${technicalLevelPickerTop(option.Titles)}
+    ${technicalLevelPickerBottom({})}
+    `
+}
+
+const technicalLevelPicker = function (option = {}){
+    return `<div class="radio-boxes">
+   ${constructorTechnicalLevelPicker(option)}
+    <br>			
+    </div>
+`
+} 
+
+const textAreaBox = function(option = {}){
+    return `<div class="user-textarea">
+        <h3>${option.HeaderTitle}</h3>
+        <textarea placeholder="${option.Placeholder}"required></textarea>
+    </div>`
+}
+
+
+const textArea = function (option = {}){
+    const result = []
+    for(var i=0;i<option.HeaderTitle.length;i++){
+        result.push(textAreaBox({HeaderTitle: option.HeaderTitle[i], Placeholder: option.Placeholder[i]}))
+    }
+    return result.join('')
+}
+
+
+const selectConstructor = function(option= {}){
+    const result = []
+    result.push(`<option selected disabled hidden>Evaluation</option>`)
+    for (var i=0;i<option.Values.length;i++){
+        result.push(`<option value = "0"> ${option.Values[i]}</option>`)
+    }
+    return result.join('')
+}
+
+const dropDownConstructor = function (option={}){
+    const result = []
+    for (var i=0;i<option.selectName.length;i++){
+        result.push(`
+        <li class="legend-box-drop">
+            <label for="${option.selectName[i]}"> ${option.labelTitle[i]}</label>
+            <select name="${option.selectName[i]}">
+            ${selectConstructor(option.sContent)}
+            </select>
+        </li>
+        `)
+    }
+    return result.join('')
+}
+
+const newEvaluationForm = function (option = {}){
+    var result = []
+    for(var i=0;i<option.Data.length;i++){
+        result.push(`<form class="legend-box" id="${option.boxID[i]}">
+        <fieldset>
+            <legend>${option.Title[i]}</legend>
+            <ul class="legend-box-list">
+                ${dropDownConstructor(option.Data[i])}
+            </ul>
+        </fieldset>
+    </form>`)
+    }
+    return result.join('')
+}
+
+const dropDownSelector = function (option = {}){
+
+}
+
+const submitButton =function (option = {}){
+    return `
+    <div class= "submit-button">
+        <form action="Evaluation.html">
+            <input  type="submit" name="button" value="Submit">
+        </form>
+    </div>`
+}
+
+const Footer = function(option = {}){
+    return `	
+    <footer class="footer footerT" >
+        <p>${option.Footer}</p>
+    </footer>`
+}
+
+const PageGrid = function (option = {}){
+    return `	<div class = "mainBlock">
+    ${candidateDetailsForm(option.cContent)}
+    ${technicalLevelPicker(option.tlContent)}
+    ${textArea(option.taContent)}
+    ${newEvaluationForm(option.sContent)}
+    ${submitButton()}
+    </div>
+    `
+}
+const NewEvaluationPage = function(option = {}){
+    return `${NAV(option.hContent)}
+    ${PageGrid(option)}
+    ${Footer(option.fContent)}`
+}
+
+
+window.onload = function(){
+    const result = [];
+    headerContent = {Logo : "/assets/logo-v2.jpg",
```

Comment:
This is a pretty large config as is, but compared to the mammoth that was the original HTML, not too shabby, is it? :P